Library/Homebrew/cmd/help.rb

```diff
@@ -1,18 +1,17 @@
 HOMEBREW_HELP = <<-EOS
 Example usage:
+  brew search [TEXT|/PATTERN/]
```

Comment:
`/REGEX/` works equally well for me and I'm happy to make that change. Has anyone else an opinion on that?